From 7091fe54a6afdda7e19fb38e115cdb7cedebb0d7 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 13 Jun 2014 11:01:33 -0400
Subject: [PATCH 104/119] systemd: give plymouth-quit-wait a better description

plymouth-quit-wait.service is a synchronization point that
other services tie into, so they know when boot is finished
and the splash is down.

Unfortunately, in cases where the boot stalls, the last thing
users see is:

Starting Wait for Plymouth Boot Screen to Quit...

and then assume the problem is with the boot splash.

This commit makes the description less finger-pointy.
---
 systemd-units/plymouth-quit-wait.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/systemd-units/plymouth-quit-wait.service.in b/systemd-units/plymouth-quit-wait.service.in
index 1c431b6..89d9004 100644
--- a/systemd-units/plymouth-quit-wait.service.in
+++ b/systemd-units/plymouth-quit-wait.service.in
@@ -1,5 +1,5 @@
 [Unit]
-Description=Wait for Plymouth Boot Screen to Quit
+Description=Hold until boot process finishes up
 After=rc-local.service plymouth-start.service systemd-user-sessions.service
 
 [Service]
-- 
1.9.0

